recipe/meta.yaml

```diff
@@ -20,18 +20,24 @@ requirements:
     - python >=3.8
     - pytest-runner >=5.2
   run:
-    - aicsimageio >=4.1
+    - aicsimageio >=4.6.3
     - fsspec
-    - napari >=0.4.11
-    - napari-plugin-engine >=0.1.4
+    - napari >=0.4.13
+    - psutil >=5.7.0
     - python >=3.8
+    # plugins
+    - bioformats_jar >=2020.5.27
+    - imageio >=2.9.0
```

Comment:
@evamaxfield ... how important was that <2.11.0 pin?  that will very likely cause problems for napari

Reply:
(note that napari has `!=2.11.0`)

Reply:
Uhhhhhh somewhat important. Maybe just remove this then? The associated bug is here: https://github.com/AllenCellModeling/aicsimageio/issues/361

Haven't gotten around to updating usage yet.

Reply:
have you tried `2.12.0`?  that 2.11 version screwed us up too, but everything since has been fine

Reply:
(but will remove for now)

Reply:
I haven't. I can try later today. ~4 hours from now ish.

Reply:
Removing imageio for now works for me!